lib/kubeclient/common.rb

```diff
@@ -224,6 +233,12 @@ def bearer_token(bearer_token)
           req['authorization'] = "Bearer #{@bearer_token}"
         end
       end
+
+      def basic_auth(user, password)
+        @basic_auth_user = user
+        @basic_auth_password = password
+        update_rest_client
```

Comment:
Don't call `update_rest_client`. How to address the bug is another discussion. Currently we have the same problem with ssl.